New abilities first commit (#36)
GameBrains/er-core: modules/experimental/pages/portal.adoc

== Game effects of {portal}

Where your army includes any units or objectives that have the {portal} ability:

* Regardless of the scenario, you may keep any detachments that contain units that can use a portal out of the play area, as <<Portal Reserves>>.

Where your unit or objective has the {portal} ability:

* When you wish to move a detachment into or out of a portal, you may choose only Advance or Assault orders.
* When you wish to move a detachment into or out of a portal, use the normal xref:main-rules:transportation.adoc[] rules to embark and disembark the units.
footnote:[
When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
]
* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
* You cannot use portals to transfer detachments *out* of the play area during the game.

> However, in the Movement phase only, you may move a detachment that is already in the play area into an available portal -- but then you *must* immediately disembark the detachment from a different available portal elsewhere in the play area.

Reviewer: This ability is relatively complex so, inevitably, I'm finding little logical gaps and edge cases as I edit.

As it's for the Experimental section we can gloss some of these questions if we get to a point where we just want to get it out there -- but in the meantime I'll note them for consideration. In this case:

- What if some units have insufficient movement or are not permitted to pass through one of the two portals? Should we leave them behind, potentially out of command? Or should they be destroyed?

Author: We're following the standard embark/disembark rules, so they'll be left behind.

= Portal

`Unit ability`
footnote:[
include::special-rules:partial$no-such-ability-in-e40k.adoc[]
]

Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
This can be an effective method to deploy reserve forces at the optimum time and place.

Well-known examples include Necron portals and Eldar Webway portals.

---

> 

Reviewer: ```suggestion

NOTE: The rules that govern how we use the {portal} ability to transport units are *almost* the same as for the {transport-x} ability.
However, there are some important differences that are necessary to maintain game balance.

```

footnote:[
include::special-rules:partial$no-such-ability-in-e40k.adoc[]
]

Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
This can be an effective method to deploy reserve forces at the optimum time and place.

Well-known examples include Necron portals and Eldar Webway portals.

---

== Data sheet variations of {portal}

Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, as attributes in brackets.
For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.

Mostly, these attributes indicate what other friendly units can actually pass through the portal -- that is, which kinds of unit that portal permits you to embark/disembark during the game.

* `_(INF)_`: Permits Infantry units.
* `_(4+)_`: Permits Vehicle units that have an Armour value of `4+` or lower.
* `_({walker})_`: Permits Vehicle units that have the {walker} ability, regardless of their Armour value.

We also indicate how the portal affects the arrival of detachments that you intend to use with it, through the `_(Roll)_` or `_(Auto)_` attributes.
See <<Portal Reserves>> for more about this.

> TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
> They are mutually exclusive, for both thematic and practical reasons -- so {portal} units in a given army list must all be of one of these types, never a mix of both.

Author: As these will be mutually exclusive at the army list level rather than the player's choice level, I think it would be safer (and less confusing) to say 'will all be' rather than 'must all be'. So:

```suggestion
TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
They are mutually exclusive, for both thematic and practical reasons -- so {portal} units in a given army list will all be of one of these types, never a mix of both.
```

Reviewer: Hmmm. Let's break this down a bit:

- 'must' works for all time frames, but does make more sense from the point of view of the designer than the player, I agree
- 'will' implies a **future** where the army lists will exist but might not yet — it's still rather beholden to the designer's point of view
- 'are' would be more clear that we create the lists with this limitation in mind, and so "that's how you, the player, will find them", and is clearly pertinent for both existing and future lists

So, perhaps something like this:

```suggestion
TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
They are mutually exclusive, for both thematic and practical reasons -- so all {portal} units in a given army list are of one type, never a mix of both.
```

Author: 👍

== Game effects of {portal}

Where your army includes any units or objectives that have the {portal} ability:

* Regardless of the scenario, you may keep any detachments that contain units that can use a portal out of the play area, as <<Portal Reserves>>.

Where your unit or objective has the {portal} ability:

* You can use this unit to move portal reserves detachments into play after they arrive.
You can also move detachments that are in play between one portal and another.
See <<portal-moves>>.

[[portal-moves]]
== Use a {portal} unit to move your forces

You can use portals to move some of your forces around.
Where you wish to move one of your detachments into or out of a portal, and it contains permitted units:

* You may do this only in the Movement phase.

> // TODO: Is this stipulation necessary? On the one hand we're trying to apply standard transport rules for simplicity, but on the other hand this kind of stipulation sets up differences from those rules which complicates things again and creates logical gaps.

Author: Yes, it's 100% necessary! 

If detachments can move through Portals in the Assault phase then they can be in a position of complete safety during the Shooting phase and then charge through the Portal, with no opportunity to shoot them first. This breaks a fundamental part of the game, where you get to shoot assaulters before they assault, unless the assaulters are exceptionally fast or have outmanoeuvred you enough that they can hide behind terrain.

In the same way, a breaking detachment could retreat through a Portal to a safe part of the table, even if they were completely surrounded by enemy units.

Reviewer: Okay, that makes sense to me.

The difficulty I'm having is that on the one hand you're saying that this is all just standard transportation rules, but on the other hand as I edit I find little logical problems that seem to imply caveats or exceptions that aren't explicit.

In this case, given what you've just explained, we effectively seem to have "just like the standard transport rules... except that you can't use it in the Assault phase". Such an exception is fine in principle, but I have to find, dissect and understand it so that I can edit effectively and make it more explicit and unambiguous in the text.

We might ultimately need to add a statement somewhere to the effect that "this works just like the standard transportation rules, with these exceptions and clarifications: [...]"

Author: 'You may do this only in the Movement phase.' is explicit isn't it?

Reviewer: > 'You may do this only in the Movement phase.' is explicit isn't it?

In itself, yes — but the point that I'm trying to make is that at present it demands close attention to figure out the extent to which embarking/disembarking a portal does or does not actually play out "just like _Transport (X)_". If I'm finding it tricky as a technical writer then it's likely to be tricky for others.

Given that they're **close** but not quite **exactly** the same in practice, I believe that we'll need to make it easier to compare and contrast so that the reader can grasp the differences quickly. How we can do that should become clear by the time we've finished this round of editing with comment threads like this one to clarify specific points.

Reviewer: For now at least, I'll insert a note near the top of the topic to mention the fact that these things are similar but not quite the same. At least that will give players fair notice.

So, at the same time I'll remove this TODO note and resolve this thread.
```suggestion
```

* You may do this only if you give the detachment Advance or Assault orders.
* Use the normal xref:main-rules:transportation.adoc[] rules to embark and disembark the detachment's units.
footnote:[
When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
]

> // TODO: Do we need to mention something to the effect that portal reserves detachments either (a) already count as embarked so don't pay 5 cm penalty to embark, or (b) need to pay the penalty to embark, just as units already in play need to do? Also, given that for a detachment that is already in play you must embark and then immediately disembark from another portal, won't this get a bit complicated given that you must also pay a 5 cm penalty for *each* portal unit...? 

Author: For the first part, I think some nuance has been lost in the reorganisation. This section reads as if it only covers moving into one Portal and out of another, with no references to disembarking when arriving from Portal Reserves.

The Portal Reserves section only mentions disembarking when discussing the edge case where some of the units won't fit through a Portal.

For the second part, I think I'm missing something that makes it complicated. You're paying a 5 cm penalty for each Portal unit as you're embarking on one and disembarking from another, instead of paying 2 x 5 cm for embarking and disembarking from the same unit.

Reviewer: > For the first part, I think some nuance has been lost in the reorganisation. This section reads as if it only covers moving into one Portal and out of another, with no references to disembarking when arriving from Portal Reserves.

Okay, I'll get my head back into it and look at that again soon. I think I see what you mean. I was trying to unpick some other issues, and this section is indeed phrased in terms of moving detachments that are already on the table, to clarify that side of things. But I can see that there's a bit missing on how to disembark from Portal Reserves.

> You're paying a 5 cm penalty for each Portal unit as you're embarking on one and disembarking from another, instead of paying 2 x 5 cm for embarking and disembarking from the same unit.

That's not what the Transportation rules state. They state that you pay 5cm to embark, and you pay 5cm to disembark — so if you do both then you pay 2x5cm=10cm. Again, is this "just like the standard Transportation rules", or not? 

Further, we have the particular case of arriving from Portal Reserves: If it should be exactly like standard Transportation rules then it still leaves room for debate on whether or not they count as 'already embarked'. I think our options are, in correlation with my TODO note here:

a. Make it clear that for units arriving from Portal Reserves the player must both embark and disembark and thus pay 2x5cm=10cm.
b. Make it clear that Portal Reserves count as already embarked and thus the player only has to disembark and thus pay 1x5cm=5cm.

I believe that (b) would be more consistent with how most people probably play when they deploy units that arrive in regular _Transport (X)_ units. But people may have different mental models for portals and Portal Reserves, and we should be take care to remove the room for debate.

Author: > That's not what the Transportation rules state. They state that you pay 5cm to embark, and you pay 5cm to disembark — so if you do both then you pay 2x5cm=10cm. Again, is this "just like the standard Transportation rules", or not?

I was replying to:

> won't this get a bit complicated given that you must also pay a 5 cm penalty for *each* portal unit...?

Which appeared to be talking about _the Portal units_ not the cargo units, so I explained the effect on the Portal units. 

Overall I think we're saying the same thing in different ways:

- Portal unit A pays 5 cm because the cargo units embark.
- Portal unit B pays 5 cm because the cargo units disembark.
- The cargo units pay 10 cm because they embark and disembark.

> Further, we have the particular case of arriving from Portal Reserves: If it should be exactly like standard Transportation rules then it still leaves room for debate on whether or not they count as 'already embarked'. I think our options are, in correlation with my TODO note here:
> 
> a. Make it clear that for units arriving from Portal Reserves the player must both embark and disembark and thus pay 2x5cm=10cm. b. Make it clear that Portal Reserves count as already embarked and thus the player only has to disembark and thus pay 1x5cm=5cm.
> 
> I believe that (b) would be more consistent with how most people probably play when they deploy units that arrive in regular _Transport (X)_ units. But people may have different mental models for portals and Portal Reserves, and we should be take care to remove the room for debate.

It's option (b). If you think a clarification is needed, please include one. I didn't specify in the original text because it was included in the way the Portal Reserves section was structured.

Reviewer: > I was replying to:

>> [Also, given that for a detachment that is already in play you must embark and then immediately disembark from another portal,] won't this get a bit complicated given that you must also pay a 5 cm penalty for each portal unit...?

Yeah, I'd gotten slightly lost in the convolutions. A brief example to explain what I meant by 'complicated' in this context:

1. You move a detachment the contains {portal} unit A.
2. You move some other detachments.
3. You move a detachment that contains {portal} unit B.
4. You move detachment X with the intent to embark at B and disembark at A. However, did you think far enough ahead to leave both A and B with enough unused movement to 'pay' the 5 cm cost for each? Just how much did you move unit A five or ten minutes ago?

Assuming I have the correct interpretation, this seems relatively complex to me. No doubt some people's minds are untroubled by the amount of thinking ahead and memory that this involves, but that's not universally true — I'm pretty sure I'd find it troublesome. Hence I queried it.

If you disagree and don't think that there is cause for concern then we can go ahead with it and see what happens — the more so as this topic is 'experimental' anyway.

---

> It's option (b). [b. Make it clear that Portal Reserves count as already embarked and thus the player only has to disembark and thus pay 1x5cm=5cm.] If you think a clarification is needed, please include one. I didn't specify in the original text because it was included in the way the Portal Reserves section was structured.

I checked the draft as it was before I started editing, and I can't see how it was 'included'. [History — state of this file at commit 0b60f9d, just before I started editing](https://github.com/GameBrains/er-core/blob/0b60f9da7020d42d27de70b5518f1690670abad4/modules/experimental/pages/portal.adoc). You might argue that it's sort of implicit in the way that the text only mentioned disembarking, but that's a bit obscure in my opinion.

The thing that got me thinking in the first place boils down to this I think:

* For standard Reserves, I believe that we tend to picture the cargo units as assigned to a particular set of transports, and perhaps already in transit, and therefore already embarked by default.
* For Portal Reserves, we state that you can choose which portal to use to put a detachment into play. So, you might not picture the state of cargo units in quite the same way. You _might_ picture them mustering somewhere, near the 'entry' portal. When you decide which portal they will use as the 'exit' portal to disembark, that's when they actually embark at the entry portal — which could imply a cost to embark as part of the overall move. That's not the only way to think of it, but it's the image that popped into my mind as 'most obvious', and thus why I raised the question.

However, I may be overthinking this. I'll look again and maybe add something to make 'already embarked' explicit, or maybe skip it and return later if people actually raise questions about it.

Reviewer: I'll go with this for now. Maybe we'll drop it later, or maybe future drafts will make it redundant anyway.

```suggestion
footnote:[
When you disembark units that you're bringing into play from Portal Reserves, treat them as already embarked -- so you only pay the 5 cm cost to disembark.
]
```

* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
Any portal that you use becomes unavailable until the start of the next turn.

> * You may move a detachment that is already in the play area into an available portal -- but only if you immediately disembark the detachment from a different available portal elsewhere in the play area.
> You cannot use portals to transfer detachments *out* of the play area during the game.

Author: Could this be simplified? You're adding an extra clause to cover something that can't happen, by inferring that there is a choice. Maybe something this instead?

```suggestion
* You may embark a detachment that is already in the play area into an available portal -- but you *must* immediately disembark the detachment from a different available portal elsewhere in the play area.
```

Reviewer: Hmm. I was trying to pre-empt the question and make it really clear that players can't use portals to take units out of the play area — especially given that transport aircraft **do** allow this. (Again, we can view this as a difference from standard transportation rules, or at least a difference in how they apply to portals as compared with transport aircraft.)

I think that your suggestion works, and it's good to get the word 'embark' in here — but I wonder if it still makes sense to include the distinct point I was trying to make somewhere (perhaps as a note or footnote instead)?

Author: If you think it's needed.

I don't think it is, because the revised suggestion only gives you a single option for what happens when a unit embarks into a Portal.

Hmm, that reminds me, it's units, not detachments. Revised suggestion:

```suggestion
* You may embark a unit that is already in the play area into an available portal -- but you *must* immediately disembark the unit from a different available portal elsewhere in the play area.
```

Reviewer: > If you think it's needed.
> I don't think it is, because the revised suggestion only gives you a single option for what happens when a unit embarks into a Portal.

I take your point, in line with the principle that if the rules don't actually say you can do something then you should assume that you can't. But I lean towards making it explicit that we do indeed intend that there is no option, at any point, to use a portal to remove units from the play area, and that it's not just something that we neglected to mention at this particular point. The more so given that:

* This is in contrast to transport aircraft.
* This is in contrast to the NetEA Necron portal rules for Epic Armageddon, which may be more familiar to some. (Very much a low priority consideration, but still, another small point in favour of being explicit.)

---

> Hmm, that reminds me, it's units, not detachments. Revised suggestion:

It's tricky because in this topic we often need to speak of putting _detachments_ into play, being able to use a given portal only once per turn to embark or disembark the units of one _detachment_, and so on.

This one is perhaps arguable, but I'm happy to go with your suggestion here.

---

So, putting both bits together here:

```suggestion
You may embark a unit that is already in the play area into an available portal -- but you *must* immediately disembark the unit from a different available portal elsewhere in the play area.
footnote:[
To be clear, there is no option to use a portal to transfer units *out* of the play area during the game.
]
```

== Portal Reserves

These Portal Reserves rules are a special variation on the standard Reserves rules.
The difference is in the detail of how detachments that you set aside as portal reserves will arrive, and how you put them into play.

> IMPORTANT: You can put portal reserves detachments into play *only* by means of {portal} units.

Author: Following on from the earlier comment, I think this needs to mention 'disembarking' in some way.

Maybe this?

```suggestion
IMPORTANT: You can put portal reserves detachments into play *only* by disembarking from {portal} units.
```

Reviewer: I **deliberately** avoided this terminology at this point.

I framed the note in more general terms because we haven't yet made it plain that we will apply the same specific terminology (and much the same mechanisms) as we used in the standard transportation rules. That becomes clear just a little later, after this note (notwithstanding any further work we need to do to improve the draft).

It's not critical, most people will get it without being thrown off, but on the whole I'd argue that it's better to stick to the more general phrasing for the more general point that comes relatively early, and only switch to the specific phrasing once we've made it plain in this context and we're talking about the specifics.

Author: I'll leave it to you, as you have final say and we have different approaches to this kind of thing.

Personally I try to only use general phrasing if I need to cover multiple options. If there is only one option you're swapping phrasing partway through the text, for no benefit in readability.

Reviewer: > I'll leave it to you, as you have final say and we have different approaches to this kind of thing.

👍I appreciate that. But just so you know, I don't think our approaches are so different here:

> Personally I try to only use general phrasing if I need to cover multiple options. If there is only one option you're swapping phrasing partway through the text, for no benefit in readability.

I agree with this in general. But in this particular case, at this particular point in the text, your suggestion would mean that we'd use a term that has specific connotations before we've actually introduced the term with its connotations for this context (that happens a few paragraphs later). The way I've approached it is **progressive disclosure**, not 'swapping phrases'.

No biggie, just thought I'd elucidate my rationale in case it helps with similar cases later. You're certainly right to keep an eye out for needless changes in how we phrase particular rules concepts, similar to how we avoid synonyms for significant terms.

Reviewer: Actually, I'm talking half sense and half out of my arse — progressive disclosure is valid, but the text does introduce embark/disembark terms higher up the page... This topic and the shuffling around made my brain hurt, and I lost track a bit! 😂

Anyway, I don't think it matters much — we aren't using needless synonyms, which is definitely bad, and I don't think that general phrases like 'put into play by means of' don't really detract from or confuse more specific use of the terms that define those 'means'.

It might change some more anyway by the time we complete this round of editing. 😅 (Although, we're on the home straight now I think, and worst case we could live with a few open questions on this experimental topic.)

* Detachment A, which contains some Necron Warriors (`INF`), Lokhust Destroyers (`INF`), and Tomb Blades (`VEH` with Armour `4+`).
* Detachment B, which contains some Necron Immortals (`INF`), Canoptek Spyders (`VEH` with `{walker}`), and a C'Tan Shard (`VEH` with `{walker}`).

As the portals in the Necron army list are `_(Auto)_` portals, detachments A and B will both arrive in turn 2.

When they arrive, you can use detachments A and B with the {portal} units as follows:

* All of the units in detachment A can pass through either portal.
So, you can use either {portal} unit to disembark detachment A, provided the portal is available.
* The {walker} units in detachment B can pass through only the Warbarque's portal.
So, you must either:
** Use the Warbarque {portal} unit to disembark this detachment, provided the portal is available.
** Use the Monolith {portal} unit to disembark this detachment, provided the portal is available -- but in this case, the Canoptek Spyders and the C'Tan Shard are *destroyed* and you must place a Blast marker on the Immortals units that form what's left of the detachment.

> // TODO: In this specific example, assuming both portals are available in turn 2, can the player deliberately choose to take a loss and disembark detachment B through the Monolith's portal and A through the Warbarque's portal?

Author: I can't see anything that would stop it, and you've covered it in your example.

Reviewer: Cool. In that case I think we can simply remove my query.
```suggestion
```

* Where your army has `_(Auto)_` portals, do *not* roll on the Reserve Detachment Arrival table for your Portal Reserves detachments in the Start phase -- instead, they always arrive in turn 2.

Either way, once a Portal Reserves detachment arrives, you *must* put it into play if possible, in the Movement phase.

=== Put Portal Reserves detachments into play (or not!)

This can be tricky.
To put a Portal Reserves detachment into play after it arrives, you need an available portal -- and some units can only use particular portals that permit their transit.

* If there is no portal available when you need it, then you must leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.^*^
* If none of the available portals can disembark all of the units in the detachment, then you may choose either:
** Leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.^*^
** Disembark as much of the detachment as you can.
The units that cannot disembark count as destroyed for the purposes of army morale and victory conditions.

> // TODO: Again -- are they actually destroyed, or only 'count as destroyed', for the sake of campaigns?

Reviewer: Remove TODO note that is now redundant.
```suggestion
```

Place a Blast marker on the detachment.

^*^If you leave the detachment in Portal Reserves, then:

* If your army uses `_(Roll)_` portals, then you must roll on the Reserve Detachment Arrival table again next turn.
* If your army uses `_(Auto)_` portals, then you must try again next turn.

> IMPORTANT: If the detachment does not arrive and enter play before all {portal} units/objectives in your army are destroyed or the game ends, then the detachment counts as destroyed for the purposes of this game.

Reviewer: Align with tweak we made a little further up:
```suggestion
IMPORTANT: If the detachment does not arrive and enter play before all {portal} units/objectives in your army are destroyed or the game ends, then the detachment counts as destroyed for the purposes of army morale and victory conditions.
```

== Game effects of {portal}

Where your army includes any units or objectives that have the {portal} ability:

* Regardless of the scenario, you may keep some detachments out of the play area, as <<Portal Reserves>>.
You may do this with any number of your detachments that contain units that your {portal} units permit as cargo.
+
TIP: Don't get carried away -- if you have many Portal Reserves detachments out of play and few suitable {portal} units in play then you're in for a hard time!

Where your unit or objective has the {portal} ability and is in play:

* You can use this unit to move Portal Reserves detachments into play after they arrive.
See <<Portal Reserves>>.
* You can use this unit to move detachments that are in play from this portal to another portal that is in play, or from another portal that is in play to this portal.
See <<portal-moves>>.

[[portal-moves]]
== Use a {portal} unit to move your forces

You can use portals to move some of your forces around.
Where you wish to move one of your detachments into or out of a portal, and it contains permitted cargo units:

> * You may do this only in the Movement phase.
> * You may do this only if you give the detachment Advance or Assault orders.
> See xref:main-rules:transportation.adoc[].
> footnote:[
> When you disembark cargo units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
> In this case, just deduct 5 cm from the movement of each cargo unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
> ]

Reviewer: I'm not sure why I put the Transportation topic reference in the same paragraph as the bit about Advance or Assault orders — that seems odd, because it implies that that topic has something to say about those orders, which it doesn't. 

I probably meant to put such a link _somewhere_ to reference the rules for embarking and disembarking with with the {transport-x} ability.

```suggestion
* You may do this only in the Movement phase, and only if you give the detachment Advance or Assault orders.
* Embark and disembark cargo units in the same way as for the {transport-x} ability.
See xref:main-rules:transportation.adoc#_transportation_by_transport_x_units[Transportation by Transport (X) units].
footnote:[
When you disembark cargo units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
In this case, just deduct 5 cm from the movement of each cargo unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
]
```

// TODO: Do we need to mention something to the effect that portal reserves detachments either (a) already count as embarked so don't pay 5 cm penalty to embark, or (b) need to pay the penalty to embark, just as units already in play need to do? Also, given that for a detachment that is already in play you must embark and then immediately disembark from another portal, won't this get a bit complicated given that you must also pay a 5 cm penalty for *each* portal unit...? 

> * You may use each individual portal only *once per turn*, to embark or disembark a single detachment.

Reviewer: ```suggestion
* You may use each individual portal only *once per turn*, to embark or disembark units from a single detachment.
```

= Portal

`Unit ability`

Some races, such as Necrons and Eldar, can open portals directly onto the battlefield.

---

[NOTE]
====
This ability lets you keep detachments off-table at the start of the game, similar to Reserves.
You can then deploy them later in the game -- when they arrive, you disembark them into the play area and have them through a unit -- or objective -- that has this ability.
====

== Portal Reserves

Where your army includes any units -- or objectives -- that have the {portal} ability:

> * Regardless of the scenario, you may keep any detachments that can use a {portal} off-table.

Reviewer: What about when the detachment is already in play and you embark through one portal and immediately disembark through another, as mentioned elsewhere in this topic? Can we then use March orders as well, or must we still use only Advance or Assault orders?

Reviewer: Ugh... I'm working locally, and tried adding this comment in VS Code — but it's posted here in the wrong context (I guess it matched the line number, which has entirely different content to my local version).
I'll delete this and another similar comment I made, and keep them in the local code for now.

Where your unit or objective has the {portal} ability:

* Detachments that you place in {portal-reserves} *must* move onto the table through a {portal}.
* When you wish to move the units in a detachment out of a portal, use the normal disembarkation rules as at xref:main-rules:transportation.adoc[].
* When you disembark a detachment from a portal, you may choose only Advance or Assault orders.
// TODO: Can you use any order you like when you disembark immediately after embark elsewhere in the play area as mentioned in the point a little further down?
* You may only use each portal to embark or disembark a single detachment in each turn.
* In the Movement phase, you may embark friendly units 'into' a portal -- but then you *must* immediately disembark them from a different {portal} elsewhere in the play area.
* Data sheets state the units that can embark/disembark through the portal in brackets, for example:
** `_(INF)_` allows Infantry units to use the {portal}.
** `_(Walker)_` allows Vehicle units that have the {walker} special ability to use the {portal}.
** `_(4+)_` allows Vehicle units that have an Armour value of `4+` or lower to use the {portal}.
** Often, we combine these.

> For example: `_(INF, {walker}, 4+, Auto)_`.

Reviewer: This seems a bit strange to me: First, it seems illogical that units that didn't participate in the battle are automatically destroyed just because they are running late. 'Count as destroyed' for some game purposes would be one thing, but 'are destroyed' is another. Second, I wonder what you intend for Army Morale -- if all portals are destroyed, and thus all remaining Portal Reserves are destroyed, I believe this implies that we must deduct their Morale points at the end of the next Rally phase -- is this correct/what you intend?

= Portal

`Unit ability`

Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
This can be an effective method to deploy reserve forces at the optimum time and place.

Well-known examples include Necron portals and Eldar Webway portals.

---

[NOTE]
====
This ability lets you keep detachments outside of the play area at the start of the game, similar to Reserves.
You can then deploy them later in the game.

> When they arrive, you disembark them into the play area through a unit -- or objective -- that has this ability.

Reviewer: How will we apply this to an objective? Will there be a new type of objective especially for this, alongside the likes of Take and Hold?

Author: To be discussed on FB. This part is future-proofing for potential Eldar Webgates.

Reviewer: Got it — now that I know Auto and Roll are mutually exclusive for a given army, it's making more sense. I'll make some tweaks so that these factors are more obvious.

Of course, for content in the experimental section we can cut some corners if you just want to get it out there — but I've defaulted to giving this much the same attention as I've given to the 'official' content that forms the bulk of this PR, since we're not rushing this through anyway.

====

== Data sheet variations of {portal}

Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, in brackets.
For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.

Mostly, this indicates what other friendly units can actually use the portal -- that is, which kinds of unit you can embark/disembark through that portal during the game.

* `_(INF)_` allows Infantry units to use the {portal}.
* `_(4+)_` allows Vehicle units that have an Armour value of `4+` or lower to use the {portal}.
* `_(Walker)_` allows Vehicle units that have the {walker} ability to use the {portal}, regardless of their Armour value.

We also indicate how the portal affects the arrival of detachments that use it:

* `_(Auto)_` means that detachments in Portal Reserves do *not* follow the standard rules for Reserves.
Instead, from turn two onwards, you must *must* disembark them from a {portal} unit/objective in the Movement phase unless there are not enough portals available, or none of the available portals can disembark all of the units in the detachment.

> * `_(Roll)_` means that detachments in Portal Reserves follow the standard rules for Reserves.

Reviewer: _`(Auto)`_ and _`(Roll)`_ are particular to specific {portal} units -- but I can't see how the respective 'follow/don't follow standard Reserves rules' effects could each apply to only some detachments and not others. Is this a gap in the current logic, or am I missing something?

Author: The quick answer is that they would never appear in the same list!

So _`(Auto)`_ covers the Necrons who are sitting waiting on the other end of the Portal, while _`(Roll)`_ is future-proofing for Eldar making their way through the Webway.

Reviewer: Aaaaah, I see... I didn't get that at all. Okay, we should tweak things a bit such that it is plain that these attributes are in effect mutually exclusive and reflect different technologies in different armies. I'll play around with that.

When a detachment arrives from Portal Reserves, you *must* disembark it from a {portal} unit/objective, if possible.

== Game effects of {portal}

Where your army includes any units or objectives that have the {portal} ability:

* Regardless of the scenario, you may keep any detachments that can use a portal out of the play area, as <<Portal Reserves>>.

Where your unit or objective has the {portal} ability:

* When you wish to move the units in a detachment out of a portal, use the normal disembarkation rules as at xref:main-rules:transportation.adoc[].
footnote:[
When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
]

> * When you wish to embark or disembark a detachment through a portal, you may choose only Advance or Assault orders.

Reviewer: Do we need to state something to the effect that Portal Reserves are already embarked, so there is no need to deduct from their movement for embarkation?

Author: If you think it's needed. I didn't mention it because when the detachment arrives on the table it's only disembarking.

Reviewer: Okay. I think it _might_ be needed, depending on how the overall phrasing works out in comparison to texts that relates to the standard Reserves and transportation rules.

Perhaps I'm over-thinking it, but there were one or two subtle points that made me wonder if some players **might** suppose that the first move from off-table implicitly involves an embark action in a way that regular transportation doesn't -- to enter an imaginary off-table portal before the explicit disembark action to exit the on-table portal, and thus **two** 5 cm deductions to the first move.

By the time I've rephrased and restructured bits, that possibility might diminish anyway; but if it still seems iffy then I'll add something explicit to clarify.

* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
* In the Movement phase, you may embark friendly units 'into' a portal.
But then you *must* immediately disembark them from a different {portal} elsewhere in the play area -- you may not place them outside of the play area again, so they cannot become Portal Reserves again.

== Portal Reserves

Apply these rules to each detachment that you have outside of the play area as Portal Reserves.

When the detachment should arrive, you must put it into play if possible -- but you may only put it into play through a suitable, available {portal}.

* If there is no portal available when you need it, then you must leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.
You must roll again next turn.
* If none of the available portals can disembark all of the units in the detachment, then you may choose, either:
** Leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.

> You must roll again next turn.

Reviewer: Confusing -- the roll isn't relevant where _`(Auto)`_ is in effect, but it's also unclear how this actually _goes_ into effect alongside _`(Roll)`_.

Author: This appears to have got a bit garbled during editing. 

I had two separate bullet points, one for _`(Roll)`_ and one for _`(Auto)`_, so 'roll' was never mentioned for _`(Auto)`_.

Author: The Roll bullet said:

* If the {portal} lists 'Roll' in brackets, detachments in {portal-reserves} follow the standard rules for Reserves. When a detachment arrives from {portal-reserves}, it **must** disembark from a {portal}. If there are not enough Portals available, or none of the available Portals can disembark all the units in the detachment, the detachment stays in {portal-reserves} and rolls again next turn.

Reviewer: Got it — now that I understand from your other comment that these attributes are mutually exclusive, I'll be able to figure out what to do with the edits.

** Disembark as much of the detachment as you can.

> The units that cannot disembark are immediately destroyed.

Reviewer: This seems a bit strange to me:

First, it seems illogical that units that didn't participate in the battle are automatically _destroyed_ just because they are 'running late'. 'Count as destroyed' for some game purposes would be one thing, but to imply 'are actually destroyed as a consequence of not being on time to participate in the battle' is another.

Second, I wonder what you intend for Army Morale -- if all portals are destroyed, and thus all remaining Portal Reserves are destroyed, I believe this implies that we must deduct their Morale points at the end of the next Rally phase -- is this correct/what you intend?

Author: Yes, I intend that all detachments still in Portal Reserves are destroyed, with all the usual effects on Army Morale. It's a very powerful ability, with a very powerful downside.

I don't mind whether we do that by saying that the detachments 'are' destroyed, or by saying that they count as destroyed for all gaming purposes.

Reviewer: Okay, makes sense.

One quibble: While they count as destroyed for all in-game 'battle' purposes, might it be wise to phrase in such a way that it's clear that they aren't 'destroyed/wiped out' for campaign 'war' purposes?

I'm thinking of the rules at [Are you experienced?](https://thehobby.zone/resources/e40k-compendium/Content/Battles/Campaigns/AreYouExperienced.htm), especially under 'Being wiped out' and 'Replacing casualties'.

Author: Fine by me.

Reviewer: Perhaps something like this. Might be room to streamline it some more.
```suggestion
The units that cannot disembark count as destroyed for the purposes of this battle.
footnote:[
For the purposes of an ongoing campaign, we do *not* consider these units to be destroyed.
This means that if you use rules like link:http://https://thehobby.zone/resources/e40k-compendium/Content/Battles/Campaigns/AreYouExperienced.htm[Are you experienced?^] (external URL) then their detachment is not 'wiped out'.
But as far as morale and victory conditions for the current battle are concerned, treat them  as destroyed.
]
```

Author: Streamlining suggestion:
```suggestion
The units that cannot disembark count as destroyed for the purposes of army morale and victory conditions.
]
```
That hopefully removes the need for an explanatory footnote.

Reviewer: Yeah, that seems good — I started off with the idea to leave it somewhat broad with "purposes of this battle" in case there was stuff I hadn't thought of, as well as the idea to reference the existing campaign/experience rules. But, if you're happy that this boils down to just army morale and victory conditions and there is nothing else, then your more minimal suggestion works and I can live without the reference to those other rules.

Reviewer: A tiny tweak to your suggestion, just so that we don't insert a redundant square bracket.
```suggestion
The units that cannot disembark count as destroyed for the purposes of army morale and victory conditions.
```
